components/Dialogs/README.md

```diff
@@ -210,11 +224,18 @@ the design parameters that you can use to theme your dialogs.
 // Step 1: Import the DialogThemer extension
 import MaterialComponents.MaterialDialogs_DialogThemer
```

Comment:
Regenerated Dialogs doc to bring color and typography themers back.

Reply:
Looks like this wasn't restored correctly when you regenerated the documentation. It might be easier to put the new documentation as a "sibling" section to the Themers documentation.

Reply:
What I mean is that it looks like you're removing the old docs from the snippet. You include the old Podspec and the old import, but you don't keep the actual themer code.  Am I reading the wrong, or would you please put the Theming Extensions docs into a separate section of the README?